packages/agent/web-test-runner.config.cjs

```diff
@@ -24,7 +24,7 @@ module.exports = {
   concurrentBrowsers : 2,
   testFramework      : {
     config: {
-      timeout: '15000',
+      timeout: '30000',
```

Comment:
Are there specific tests that are taking longer than 15 seconds to run?  Curious if there is just 1 or 2 outliers.

Reply:
its the usual suspects in the `agent` test suite